src/main/webapp/i18n/en/exercise.json

```diff
@@ -189,7 +189,8 @@
                     "results": "The existing results must be recalculated!",
                     "confirmtext": "Do you really want to perform these changes?"
                 }
-            }
+            },
+            "isUsedForTutorial": "Tutor needs to review"
```

Comment:
In the current text exercise I test, the checkbox says: "Tutors need to assess this submission".  I would change it to what is shown in the picture, it's the same use case?
![image](https://user-images.githubusercontent.com/41366522/119890965-b9ba4c00-bf38-11eb-8bee-855cfd005677.png)

Reply:
Yes, you are correct, it is indeed the same use-case.
The dashboard column title is now the same key/string as shown on the screenshot.
But I renamed it a bit, by replacing the word _assess_ with _review_. This is the terminology Professor Brügge wants to be used by tutors from now on.